apps/iis.go

```diff
@@ -26,13 +26,32 @@ type MetricsReceiverIis struct {
 	confgenerator.ConfigComponent `yaml:",inline"`
 
 	confgenerator.MetricsReceiverShared `yaml:",inline"`
+	Version                             string `yaml:"receiver_version,omitempty"`
```

Comment:
Why is the field called `Version` when the user writes `receiver_version`? Shouldn't it be `ReceiverVersion`? Especially if you plan to later factor it out into the shared struct…

Reply:
Updated to ReceiverVersion.